example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -81,15 +87,26 @@ public void onActivityCreated(@Nullable final Bundle savedInstanceState) {
      * Executes all test and calls setTrustScore() to calculate an average score
      */
     public void runTests() {
+
+        SyncSecurityCheckExecutor executor = SecurityCheckExecutor.Builder.newSyncExecutor(this.getContext())
```

Comment:
The idea behind this builder structure is that you don really need to reference the Sync/Async executor. 
You can avoid saving the instance to a SyncSecurityCheckExecutor variable, and directly do:
`
Map<String, SecurityCheckResult> results = SecurityCheckExecutor.Builder.newSyncExecutor(this.getContext())
...
.build().execute();
`

Reply:
I see, so avoiding the explicit import of `SyncSecurityCheckExecutor` :+1: